tests/unit/test_workflow_id_utils.py

```diff
@@ -0,0 +1,127 @@
+import shutil
+
+import pytest
+
+from cromshell.utilities import workflow_id_utils
+
+
+@pytest.fixture
+def tmp_submission_file(mock_workflow_database_tsv, tmp_path):
```

Comment:
this seems to be a good candidate to be refactored to `conft_test.py`?

Reply:
done